### ISSUE TITLE
Source reorganization

### ISSUE THREAD

**Issue by data-man** [c3lang/c3c#383]
Proposals:
* move `resources/lib/*` to `lib/*`
* move `resources/editor_plugins/*` to another repository. This will not trigger the CI
* maybe remove `resources/*` completely?

**Comment by lerno** [comment]
Yeah, the plugins should probably be moved first. The move of lib is... I would like to think about how to organize it. Eventually it should probably be a separate project that produces a library file you just drop in.

**Comment by data-man** [comment]
And move `c3lang/intellij-plugin` to `c3lang/editor-plugins`?

**Comment by lerno** [comment]
Actual plugins that are in-app downloadable are likely to need their own repos due to how the workflows and deploy scripts are typically written. The lib/* has moved. Resources will contain example code, manuals etc.
